client/test-helpers/config/testing-library.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import { render as rtlRender } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import initialReducer from 'calypso/state/reducer';
+
+const render = ( ui, { initialState, store, reducers, ...renderOptions } = {} ) => {
+	if ( ! store ) {
+		let reducer = initialReducer;
+
+		if ( typeof reducers === 'object' ) {
+			for ( const key in reducers ) {
+				reducer = reducer.addReducer( [ key ], reducers[ key ] );
```

Comment:
We're allowing the tests consuming the custom `render` function to pass additional reducers when creating the store. This is helpful because reducers create default state parts, allowing us to not pass these parts explicitly in the initial state.